client/pages/home.jsx

```diff
@@ -1,9 +1,41 @@
 import React from 'react';
+import Post from '../components/post';
 
-export default function Home(props) {
-  return (
-    <>
-      <div className='text-3xl text-center mt-4'>HOME PAGE</div>
-    </>
-  );
+export default class Home extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      posts: []
+    };
+  }
+
+  componentDidMount() {
+    fetch('api/posts')
+      .then(res => res.json())
+      .then(posts => {
+        this.setState({ posts: posts });
+      })
+      .catch(err => console.error(err));
+  }
+
+  render() {
+    const posts = this.state.posts;
+    const shallowPosts = posts.slice();
+    const mostRecentToLeast = shallowPosts.sort((a, b) => {
+      return b.postId - a.postId;
+    });
```

Comment:
Use an `order by` clause on the server instead. You can order by timestamps just fine with Postgres.